src/zdemo_excel_checker.prog.abap

```diff
@@ -1122,7 +1120,7 @@ CLASS lcl_xlsx_diff_viewer IMPLEMENTATION.
           item_table          TYPE treemcitab,
           item_line           TYPE treemcitem,
           temp_container      TYPE REF TO cl_gui_container.
-
+    CONSTANTS: cntb_btype_button TYPE i VALUE 0.
```

Comment:
these constants from type pools can also be added to the json config, example

https://github.com/abapGit/abapGit/blob/main/abaplint.json#L22-L30

Reply:
Thanks, I didn't know that. Next time I'll do it.

Reply:
@larshp silly me, I tagged you from the commit, not sure it works...